src/byteweight/byteweight.ml

```diff
@@ -16,7 +16,7 @@ let ignored = [
 module Byteweight = Bap_byteweight.Bytes
 
 let train_on_file meth length db path : unit t =
-  Image.create path >>| fun (img,warns) ->
+  Image.create ~backend:"llvm" path >>| fun (img,warns) ->
```

Comment:
you can remove this `backend` workaround, it is already fixed (I hope) and merged